examples/android/app/BUILD.bazel

```diff
@@ -0,0 +1,47 @@
+load("@build_bazel_rules_android//android:rules.bzl", "android_binary")
+
+# An app that consumes android-kt deps
+android_binary(
+    name = "app",
+    custom_package = "examples.android.app",
+    incremental_dexing = 0,
+    manifest = "src/main/AndroidManifest.xml",
+    multidex = "native",
+    deps = [
+        "//lib",
+    ],
+)
+
+# An app that consumes jvm-kt libs
+android_binary(
+    name = "app2",
+    custom_package = "examples.android.app",
+    incremental_dexing = 0,
+    manifest = "src/main/AndroidManifest.xml",
+    multidex = "native",
+    deps = [
+        "//lib2",
+    ],
+)
+
+# An app that consumes android-kt deps, and does incremental dexing.
+android_binary(
+    name = "app3",
+    custom_package = "examples.android.app",
+    incremental_dexing = 1,
+    manifest = "src/main/AndroidManifest.xml",
+    multidex = "native",
+    deps = [
+        "//lib",
+    ],
+)
+
+sh_test(
```

Comment:
you can use `build_test` for skylib for this.

Reply:
When did that get added!  Last I saw it was an internal google thing.  I'll update.

Reply:
Done